cmd/krel/cmd/anago/push_git_objects.go

```diff
@@ -166,3 +167,14 @@ func (o *pushGitObjectsOptions) Validate() error {
 	}
 	return nil
 }
+
+// isPushableBranch returns a bool indicating if branchName should be pushed or ignored
+func isPushableBranch(branchName string) bool {
+	if branchName == "" {
+		return false
+	}
+	if strings.HasPrefix(branchName, "release-") {
+		return true
+	}
+	return false
+}
```

Comment:
We have a branch regex in the `release` package which covers this:
https://github.com/kubernetes/release/blob/85f4f2e80b9548762071e533b84d46f914f93f4c/pkg/release/regex/regex.go#L23-L27

Do we wanna use this instead?

Reply:
I've changed the function to use `regex.BranchRegex`. As it currently matches "master", I still had to filter out the main branch manually.